docs/harvesters_and_transformers.rst

```diff
@@ -162,10 +144,11 @@ Writing a source.yaml file
 The ``source.yaml`` file contains information about the source itself, and one or more configs that describe how to harvest and transform data from that source.
 
 .. code-block:: yaml
+
     name: com.example
     long_title: Example SHARE Source for Examples
     home_page: http://example.com/
-    user: providers.com.example
+    user: sources.com.example
```

Comment:
I don't think it's worth updating existing users. I just wanted to remove the term provider from the docs entirely.